profiling/src/exporter/mod.rs

```diff
@@ -156,6 +161,8 @@ impl ProfileExporter {
         files: &[File],
         additional_tags: Option<&Vec<Tag>>,
         endpoint_counts: Option<&ProfiledEndpointsStats>,
+        // See Datadog-internal "RFC: Attaching internal metadata to pprof profiles" for details
+        internal_metadata: Option<serde_json::Value>,
```

Comment:
**not blocking**
In it's current state it would accept any valid JSON. Do we know more about what the backend accepts? Like is it always only `"key: "value"` and is value only allowed to be a primitive type (string, boolean, integers, ..) or would it also accept arrays or objects?

Reply:
It also accepts arrays or objects, see the RFC document for an example where we show it in use :)